app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,15 @@
+<nav class="navbar navbar-toggleable-xl navbar-inverse bg-inverse">
+  <div class="navbar-nav">
+    <div class="container-fluid">
+      <ul class="navbar-nav mr-auto">
+        <a class="navbar-brand" href="<%= root_path %>">Image Sharer</a>
+        <li class="nav-item">
+          <%= link_to "Index", root_path, :class => "nav-link #{current_page?(root_path) ? "active" : ""}" %>
```

Comment:
Instead of turnary maybe it would be best to use a simple if statement here...
```<% if current_page?(root_path) %> active <% end %>```

Reply:
Cool.  We weren't aware of that valid syntax.  It makes logical sense, though.